apps/hash-integration-worker/src/activities.ts

```diff
@@ -1,6 +1,120 @@
-export const createIntegrationActivities = () => ({
-  // eslint-disable-next-line @typescript-eslint/require-await,@typescript-eslint/no-unused-vars
-  async helloWorldActivity(params: {}): Promise<String> {
-    return "Hello, world!";
+import {
+  Attachment,
+  Comment,
+  Connection,
+  CustomView,
+  Cycle,
+  Document,
+  Issue,
+  IssueLabel,
+  LinearClient,
+  LinearDocument,
+  Organization,
+  Project,
+  ProjectMilestone,
+  Team,
+  User,
+} from "@linear/sdk";
+
+const readNodes = async <T>(connection: Connection<T>): Promise<T[]> => {
+  const nodes = connection.nodes;
+  while (connection.pageInfo.hasNextPage) {
+    // eslint-disable-next-line no-param-reassign
+    connection = await connection.fetchNext();
+    nodes.push(...connection.nodes);
+  }
+  return nodes;
+};
+
+export const createLinearIntegrationActivities = (createInfo: {
```

Comment:
I would deconstruct this to save repeating `createInfo.linearClient` below
```suggestion
export const createLinearIntegrationActivities = ({ linearClient }: {
```